ui/app/models/pki/issuer.js

```diff
@@ -42,6 +48,59 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
   })
   uriSans;
 
+  @attr('string') issuerName;
+
+  @attr({
+    label: 'Leaf notAfter behavior',
+    subText:
+      'What happens when a leaf certificate is issued, but its NotAfter field (and therefore its expiry date) exceeds that of this issuer.',
+    dockLink:
+      'https://github.com/hashicorp/vault/blob/main/website/content/api-docs/secret/pki.mdx#update-issuer',
+    editType: 'yield',
```

Comment:
The form field component is 🥴 but if you passed `possibleValues`(instead of `valueOptions`) you'd get the select functionality for free? Plus the doclink would render via FormFieldLabel. But perhaps you had another reason for `yield`-ing, here!

The same might be true for `usage`, too

Reply:
Yes I originally tried that but the actual values are `err`, `truncate` and `permit` and the designs call for different labels used as the options. If there were an option to pass in an array of objects with a label and value or something like that it would be more flexible.

Reply:
Usage doesn't fit the pattern and I decided not to blow out the scope by adding another condition in the form field component. The property is a comma separated string which gets mapped to checkboxes.
![image](https://user-images.githubusercontent.com/24611656/212192459-f6fead11-8a25-4062-9af6-ca2a05de2798.png)

Reply:
Okay! That's what I noticed asI moved down the template file. Makes sense!